app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -306,46 +299,47 @@ class LuciBuildService {
     final List<String> builderNames = builders.map((LuciBuilder entry) => entry.name).toList();
     // Return only builds that exist in the configuration file.
-    final Iterable<Build> failedBuilds = builds.values.where((Build build) => failStatusSet.contains(build.status));
-    final Iterable<Build> expectedFailedBuilds =
-        failedBuilds.where((Build build) => builderNames.contains(build.builderId.builder));
+    final Iterable<Build?> failedBuilds = builds.values.where((Build? build) => failStatusSet.contains(build!.status));
+    final Iterable<Build?> expectedFailedBuilds =
+        failedBuilds.where((Build? build) => builderNames.contains(build!.builderId.builder));
     return expectedFailedBuilds.toList();
   }
 
   /// Sends a [BuildBucket.scheduleBuild] the buildset, user_agent, and
   /// github_link tags are applied to match the original build. The build
   /// properties from the original build are also preserved.
   Future<bool> rescheduleBuild({
-    @required String commitSha,
-    @required String builderName,
-    @required push_message.BuildPushMessage buildPushMessage,
+    required String commitSha,
+    required String builderName,
+    required push_message.BuildPushMessage buildPushMessage,
   }) async {
     // Ensure we are using V2 bucket name istead of V1.
     // V1 bucket name  is "luci.flutter.prod" while the api
     // is expecting just the last part after "."(prod).
-    final String bucketName = buildPushMessage.build.bucket.split('.').last;
-    final Map<String, dynamic> userData = jsonDecode(buildPushMessage.userData) as Map<String, dynamic>;
+    final String bucketName = buildPushMessage.build!.bucket!.split('.').last;
+    final Map<String, dynamic>? userData = jsonDecode(buildPushMessage.userData!) as Map<String, dynamic>?;
     await buildBucketClient.scheduleBuild(ScheduleBuildRequest(
       builderId: BuilderId(
-        project: buildPushMessage.build.project,
+        project: buildPushMessage.build!.project,
         bucket: bucketName,
         builder: builderName,
       ),
       tags: <String, List<String>>{
-        'buildset': buildPushMessage.build.tagsByName('buildset'),
-        'user_agent': buildPushMessage.build.tagsByName('user_agent'),
-        'github_link': buildPushMessage.build.tagsByName('github_link'),
+        'buildset': buildPushMessage.build!.tagsByName('buildset'),
+        'user_agent': buildPushMessage.build!.tagsByName('user_agent'),
+        'github_link': buildPushMessage.build!.tagsByName('github_link'),
       },
-      properties: (buildPushMessage.build.buildParameters['properties'] as Map<String, dynamic>).cast<String, String>(),
+      properties:
+          (buildPushMessage.build!.buildParameters!['properties'] as Map<String, dynamic>).cast<String, String>(),
       notify: NotificationConfig(
         pubsubTopic: 'projects/flutter-dashboard/topics/luci-builds',
         userData: json.encode(userData),
```

Comment:
You'll need to transform the encoded json string to base64.

Reply:
This was it, thank you! I'm not sure why this regressed with this PR